python-avd/pyavd/_eos_designs/structured_config/network_services/vxlan_interface.py

```diff
@@ -188,7 +188,6 @@ def _get_vxlan_interface_config_for_vrf(self: AvdStructuredConfigNetworkServices
                     underlay_l3_mcast_group_ipv4_pool_offset = get(tenant, "evpn_l3_multicast.evpn_underlay_l3_multicast_group_ipv4_pool_offset", default=0)
                     vrf_data["multicast_group"] = self.shared_utils.ip_addressing.evpn_underlay_l3_multicast_group(
                         underlay_l3_multicast_group_ipv4_pool,
-                        vni,
```

Comment:
Please do not change the signature of the function. These ip_addressing methods can be overridden by expert users with custom Python modules.

Reply:
Fixed